src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -2039,7 +2036,7 @@ public enum PrimitiveTypeCode : byte
     }
     public readonly partial struct PropertyAccessors
     {
-        private readonly int _dummy;
+        private readonly object _dummy;
```

Comment:
@weshaggard, is this change OK and expected (the _dummy fields going from int to object)?

Reply:
Some of those definitely look suspicious. object is the fallback if we cannot determine all the fields in the struct.

Reply:
Can you provide some rudimentary guidelines that we can follow to vet these changes ourselves?